condominium/__manifest__.py

```diff
@@ -0,0 +1,69 @@
+{
+    'name': 'Condominium',
+    'version': '1.0',
+    'category': 'Services',
+    'description': """
+This industry is tailor-made for condominium businesses that aim to manage co-ownership properties. Such a business is complex because it implies managing all the aspects of the properties while fairly splitting the charges.
+Moreover, the business is legally regulated in lots of countries under different names: condo in the US, co-ownership in Canada, joint domination in the UK, property syndicate in France, syndic in Belgium, etc
+""",
+    'depends': [
+        'account_check_printing',
+        'account_followup',
+        'appointment_account_payment',
+        'contacts',
+        'documents_hr',
+        'documents_product',
+        'documents_project_sale',
+        'documents_spreadsheet',
+        'helpdesk_sale_timesheet',
+        'hr_skills',
+        'sale_pdf_quote_builder',
+        'sale_planning',
+        'sale_subscription',
+        'website_appointment',
+        'website_helpdesk_knowledge',
```

Comment:
As explained in task, to me it can reduced to  
```suggestion
        'account_check_printing',
        'account_followup',
        'contacts',
        'documents_product',
        'documents_project_sale',
        'documents_spreadsheet',
        'helpdesk_sale_timesheet',
        'sale_subscription',
```